### DISCUSSION TITLE
Deploy Kotter Application to the Web?

### DISCUSSION THREAD

**Discussion by zyberzebra** [varabyte/kotter#118] [Q&A]
Is there a way to host the application in virtual Terminal on the Web? 
https://github.com/yudai/gotty and https://xtermjs.org/ are the 2 things I found searching.

TLDR: I want to show off my App to friends and thought Web would be the easiest.

**Comment by imdipp** [comment]
Reviving this one - did you get any further into getting a solution for this ?
